### DISCUSSION TITLE
How to deploy yarn 2 project properly with vercel cli?

### DISCUSSION THREAD

**Discussion by iam-yan** [vercel/vercel#5863] [Help]
I use yarn 2 pnp and deployed a project in the mono repo with vercel cli. But the depending workspaces are considered as outer libraries and thus I got the following error:
`error An unexpected error occurred: "https://registry.yarnpkg.com/@libs%2faws: Not found".`

Have tried `vc -b FORCE_BUILDER_TAG=canary` but no luck.

Help~

**Comment by ryansquadle** [comment]
Is there an update to this issue with yarn 2 pnp strategy?

**Reply by iam-yan** [reply]
fyi~ I have contacted the support but they told me so far it is not fully supported.
